templates/item/macros.j2

```diff
@@ -0,0 +1,211 @@
+{% import 'utility.macros.j2' as utility %}
+
+
+{% macro code_block(language=none, content=none) %}
+  {% set content = content if (content != none) else caller() %}
+  {% set language = 'jinja' if language == 'njk' else language %}
+  {% set language = ['lang', language]|join('-') if language != none %}
+  <pre data-sassdoc="hljs"><code{% if language %} class="{{ language }}"{% endif %}>{{ content }}</code></pre>
+{% endmacro %}
+
+
+{% macro source(item, type) %}
+{%- call code_block(language='scss') -%}
+  {%- if (type == 'variable') -%}
+    {{ scss_variable(item) }}
+  {%- elif (type == 'css') -%}
+    {{ scss_css(item) }}
+  {%- elif (type == 'function') or (type == 'mixin') -%}
+    {{ scss_function_or_mixin(item) }}
+  {%- endif -%}
+{%- endcall -%}
+{% endmacro %}
+
+
+{% macro scss_variable(item) -%}
+${{ item.context.name }}: {{ item.context.value }}{% if item.context.scope != 'private' %} !{{ item.context.scope }}{% endif %};
+{%- endmacro %}
+
+
+{% macro scss_function_or_mixin(item) -%}
+@{{ item.context.type }} {{ item.context.name }}{% if item.parameter %}(
+{%- for parameter in item.parameter %}
+  ${{ parameter.name }}{% if parameter.type|lower == "arglist" %}...{% elif parameter.default %}: {{ parameter.default }}{% endif %}{% if not loop.last %},{% endif %}
+{%- endfor %}
+){% endif %} { /* ... */ }
+{%- endmacro %}
+
+{%- macro scss_css(item) -%}
+{{ item.context.name }} {
+  {{ item.context.value }}
+}
+{%- endmacro -%}
+
+
+{# @param annotation #}
+{% macro parameter(item) -%}
+  {% if item.parameter %}
+    <div data-sassdoc="parameters">
+      <h5 class="item-subtitle">Parameters</h5>
+      <table data-sassdoc="parameters-table">
+        <thead>
+          <tr scope="row">
+            <th scope="col">Name</th>
+            <th scope="col">Description</th>
+            <th scope="col">Type</th>
+            <th scope="col">Default</th>
+          </tr>
+        </thead>
+        <tbody>
+          {%- for parameter in item.parameter %}
+          <tr data-sassdoc="parameter-item">
+            <td scope="row" data-label="name"><code>${{ parameter.name }}</code></td>
+            <td data-label="desc">{% if parameter.description %}{{ parameter.description|safe }}{% endif %}</td>
+            <td data-label="type"><code>{% if parameter.type %}{{ parameter.type }}{% endif %}</code></td>
+            <td data-label="default">{% if not parameter.default %}
+                &mdash;<span class="visually-hidden"> none</span>
+              {% else %}
+                <code>{{ parameter.default }}</code>
+              {% endif %}
+            </td>
+          </tr>
+          {%- endfor %}
+        </tbody>
+      </table>
+    </div>
+  {% endif %}
+{%- endmacro %}
+
+
+{# @require annotation #}
+{% macro require(item) -%}
+  {% if item.require.length > 0 %}
+    <div data-sassdoc="item-description">
+      <h5 class="item-subtitle">Requires</h5>
+      <ul>
```

Comment:
`<li data-sassdoc="requires-item">`, `<li data-sassdoc="link-item">`, ` <li data-sassdoc="see-item">` these types of list items need a left margin, or perhaps a class on the <ul> to include the margin, or something else to hook into like `main` `li` ... I guess the nav isn't using list items (should we be?) so maybe no specificity is needed?